Objects/stringlib/fastsearch.h

```diff
@@ -773,6 +789,9 @@ FASTSEARCH(const STRINGLIB_CHAR* s, Py_ssize_t n,
         else if (mode == FAST_RSEARCH)
             return STRINGLIB(rfind_char)(s, n, p[0]);
         else {
+            if (maxcount == PY_SSIZE_T_MAX) {
+                return STRINGLIB(count_char_no_maximum)(s, n, p[0]);
```

Comment:
```suggestion
                return STRINGLIB(count_char_no_maxcount)(s, n, p[0]);
```

Reply:
I saw it, and there was another oversight that I fixed. 

Sorry, I did it quickly just before breakfast, should have been more dilligent.

Reply:
I think you can resolve the outdated conversations.